todo_app/data/Trello_items.py

```diff
@@ -0,0 +1,75 @@
+import requests
+import os
+
+from todo_app.item import Item
+
+def add_item(title):
+    import ssl
+    context = ssl.create_default_context()
+    der_certs = context.get_ca_certs(binary_form=True)
+    pem_certs = [ssl.DER_cert_to_PEM_cert(der) for der in der_certs]
+    with open('wincacerts.pem', 'w') as outfile:
+        for pem in pem_certs:
+            outfile.write(pem + '\n')
```

Comment:
In the next module we'll introduce a `create_app` function and then it might be nice to move this code there, since we only need it to run once when we create the app. For now though, well done fighting your way past the network restrictions! 😄